ports/servoshell/desktop/app_state.rs

```diff
@@ -37,6 +38,14 @@ pub(crate) enum AppState {
     ShuttingDown,
 }
 
+/// Holds the senders for the WebDriver server.
+#[derive(Clone, Default)]
+struct WebDriverSenders {
+    pub load_status_sender: Option<(WebViewId, IpcSender<WebDriverLoadStatus>)>,
```

Comment:
I would just keep this one and remove the other two until they are used. In addition, it feels like this should be `load_status_senders: HashMap<WebViewId, IpcSender<WebDriverLoadStatus>>` in case two `WebView`s are loading at once.